custom/icds_reports/migrations/sql_templates/database_functions/update_location_table.sql

```diff
@@ -1,63 +0,0 @@
--- Update Locations Table
-CREATE OR REPLACE FUNCTION update_location_table() RETURNS VOID AS
```

Comment:
I assume there'll be a follow up PR to delete these functions

Reply:
Yes, I will send next PR when this will be merged.